src/server/multi_command_squasher.h

```diff
@@ -45,16 +46,17 @@ class MultiCommandSquasher {
   static constexpr int kMaxSquashing = 32;
 
  private:
-  MultiCommandSquasher(absl::Span<StoredCmd> cmds, ConnectionContext* cntx, bool error_abort);
+  MultiCommandSquasher(absl::Span<StoredCmd> cmds, ConnectionContext* cntx, Service* Service,
+                       bool verify_commands, bool error_abort);
 
   // Lazy initialize shard info.
   ShardExecInfo& PrepareShardInfo(ShardId sid);
 
   // Retrun squash flags
   SquashResult TrySquash(StoredCmd* cmd);
 
-  // Execute separate non-squashed cmd.
-  void ExecuteStandalone(StoredCmd* cmd);
+  // Execute separate non-squashed cmd. Return true if aborting on error.
```

Comment:
Maybe return `false` on error?

Reply:
Changed it in other places as well to conform, its a little less readable with the error_abort_ variable